### PR TITLE
New Language (Georgian / Kartuli) Cheat Sheet

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Georgian Language} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

Georgian (Kartuli) is the official language spoken in Georgia.
The cheatsheet with basic words and phrases for the same have been
created and should be included on IA goodies.

###### Description of changes
Georgian language cheatsheet JSON added

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/georgian_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@NaveenKarippai Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/georgian.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    "name": "Georgian Cheat Sheet",
+    "description": "Basic Georgian words and phrases",
+
+    "metadata": {
+        "sourceName": "101languages",
+        "sourceUrl" : "http://www.101languages.net/georgian/basics.html"
+    },
+
+    "aliases": [
+        "kartuli"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/georgian.json]: you can add more aliases by checking other language cheatsheets

File: share/goodie/cheat_sheets/json/language/georgian.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      "Going out for dinner" : [
+         {
+            "key" : "Do you have vegetarian dishes?",
+            "val" : "გაქვთ ვეგეტარიანული სადილი?",
+            "trn" : "Gaqvt vegetarianuli sadili?"
+         },
+         {
+            "key" : "I am allergic",
+            "val" : "მე ალერგიული ვარ.",
+            "trn" : "Me alergiuli var."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/georgian.json]: Remove the period at end of val field

File: share/goodie/cheat_sheets/json/language/georgian.json (hunk 1 above)
Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @jophab could you please be more specific. AFAIK, aliases are alternative keywords that would trigger the cheatsheet. For Georgian language, the alias would be "kartuli".

src: friend who is a native speaker

File: share/goodie/cheat_sheets/json/language/georgian.json (hunk 2 above)
Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @jophab Thanks for the heads up! 

**Just double checking:** wouldn't it be sufficient if I push the changeset to MY forked repo branch to update the PR?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Georgian Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/georgian_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/georgian.json (hunk 2 above)
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/georgian.json]: I think removing period at end will improve the look of the cheat sheet  slightly.
For second question,
Yes

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: hm, I prefer to keep the format as my friend who helped me with translations asked me to do it this way. I'm welcome to changes if the community asks for it! 
@jophab Thanks again!

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @NaveenKarippai Usually we prefer no period at the end of short phrases (unless it is required of the language, of course).

Also, are you a native speaker?

Comment (NaveenKarippai) [inline share/goodie/cheat_sheets/json/language/georgian.json]: @GuiltyDolphin The Georgian cheatsheet was created with the **help of a friend, who is a native speaker** (linguistic student indeed). After double checking, some words does not require (.) period at the end while others does! I shall make a PR with new changeset.

Thanks for the review!

**Comment by GuiltyDolphin** [comment]
@NaveenKarippai It looks like you're working off of master and now have a bunch of other commits! I'd suggest making another PR working off a separate branch so you don't get dodgy history!

**Comment by NaveenKarippai** [comment]
@GuiltyDolphin sorry to ask, do you want me to close this PR and open a new clean one?

**Comment by GuiltyDolphin** [comment]
@NaveenKarippai Aye!

**Pull request closed without merging**